pkg/apis/core/v1alpha1/defaults_test.go

```diff
@@ -118,11 +118,32 @@ var _ = Describe("Defaults", func() {
 		It("should default the settings", func() {
 			SetDefaults_SeedSettingDependencyWatchdog(obj)
 
-			Expect(obj.Endpoint.Enabled).To(BeTrue())
-			Expect(obj.Probe.Enabled).To(BeTrue())
+			Expect(obj.Weeder.Enabled).To(BeTrue())
```

Comment:
This test does not cover the case when `Endpoint` is set but `Weeder` is not, for example.